.circleci/config.yml

```diff
@@ -66,7 +66,7 @@ jobs:
   test:
     <<: *defaults
     docker:
-      - image: circleci/node:14-buster
+      - image: cimg/node:14.18.2
```

Comment:
Based on [the Dockerfile for the deprecated `circleci/node:stretch` image](https://hub.docker.com/layers/circleci/node/14-buster/images/sha256-cf88c16454a6fbc432f0923e2b3c291f1dd52f3cb00d41ca7af9908b4155b06e?context=explore), when upgrading to the new-gen image, I've used the same `node` version, which was `14.18.2`.